vcsh

```diff
@@ -276,6 +276,13 @@ list() {
 	done
 }
 
+list_where_remote() {
```

Comment:
Minor nit: `list_*` follows list, verb, optional. As such, could you rename to `list_has_remote`, please?